source/envs/avoidstoppedobserver.py

```
@@ -0,0 +1,341 @@
+        pygame.display.set_caption("AvoidObserver Test (ESC: Quit)")
+        clock = pygame.time.Clock()
+
+        # 상태 메시지 폰트 초기화
+        try:
+            font = pygame.font.SysFont("Arial", 24)
+        except:
+            font = None
+
+        running = True
+        while running:
+            count += 1
```

Ruff (0.12.2)

Undefined name `count`

(F821)